dashboards/resources.libsonnet

```diff
@@ -5,31 +5,6 @@ local template = grafana.template;
 {
   grafanaDashboards+:: {
 
-    local intervalTemplate =
-      template.new(
-        name='interval',
```

Comment:
I would say let’s not remove this but set the value to $__interval

Reply:
:+1: